src/main/java/seedu/address/logic/commands/CommandWord.java

```diff
@@ -20,6 +20,8 @@
     INTRO("intro"),
     LIST("list"),
     SORT("sort"),
+    REMIND("remind"),
+    REMINDER("reminder"),
```

Comment:
Confusing command names. Should change into a "add -r" and "show/list -r" style like how contacts and events are handled to keep consistency.